templates/less/components/form.less

```diff
@@ -76,3 +114,29 @@
     width: 400px;
   }
 }
+
+
+.warning {
```

Comment:
was warning independent, or does it need to be moved into label or input or something?